src/main/java/com/example/onlinebookstore/model/User.java

```diff
@@ -0,0 +1,37 @@
+package com.example.onlinebookstore.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.Setter;
+
+@Entity
+@Getter
+@Setter
+@Table
+public class User {
```

Comment:
Please use soft delete

Reply:
There is no soft delete requirement for the user entity, and in general, we do not usually delete users, and we won't have a delete endpoint for it in this project. But I think you can still leave it as it is if you want